docs/usage_as_a_library.rst

```diff
@@ -0,0 +1,24 @@
+Usage as a library
+##################
+
+Generally, the :code:`open_petro_elastic.material` module
+is used for interfacing with open_petro_elastic as
+a python library::
+
+    from open_petro_elastic.material import Material
```

Comment:
Is it an idea to use `doctest` here? https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html

Reply:
doctest is used :)

Reply:
Added!